tests/functional/centos/7/bs-lvm-osds/group_vars/all

```diff
@@ -0,0 +1,26 @@
+---
+
+ceph_origin: repository
+ceph_repository: community
+cluster: ceph
```

Comment:
can we change it to something else or `ceph-volume` does not support that yet?

Reply:
sure we can change this

Reply:
changed to 'test'